coopersystem-fsd/pcc
# Changed display order of event posts

## Description

- Changed display order of posts other than "course" or "past_course" to DESC;
- Removed limitation to not display empty post categories when creating a page with Blog Category View template.

## Steps to test

1. Check that the events page is returning events in descending order of dates and that the courses page is returning courses in ascending order of dates;
2. Create a page using the "Blog Category View" template in the desired language and check if the list of categories will appear in the language used.

## Additional information
